src/content/refi/index.md

```diff
@@ -0,0 +1,80 @@
+
+Existing technology for validating, transacting and consuming carbon credits can be improved drastically by transitioning the VCM to the new blockchain-based **digital carbon market (DCM)**. This open, transparent ecosystem ensures publicly verifiable data, access for a broad range of users and greater liquidity to scale up the amount of capital deployed to fund climate action.
+
+ReFi projects employ blockchain technology to alleviate many of the problems of the traditional market:
+
+- In the DCM, liquidity is concentrated in a small number of liquidity pools that can be freely traded by anyone. Large organizations as well as individual users can use these pools without manual searches for sellers/buyers, participation fees, or prior registration.
+- All transactions are recorded on public blockchains such as Polygon, a scaling solution for Ethereum. The path each carbon credit takes due to trading activity is traceable forever as soon as it is made available in the DCM.
+- Transaction speed is nearly instant. Securing large amounts of carbon credits via the legacy markets can take days or weeks, but this can be achieved in a few seconds in the DCM.
+- Trading activity occurs without intermediaries, which charge high fees. Data analytics firm AlliedOffsets found that digital carbon offset assets represent a [62% cost improvement compared to equivalent traditional credits](https://www.klimadao.finance/blog/klimadao-analysis-of-the-base-carbon-tonne).
+- The DCM is scalable and can meet the demands of individuals and multinational corporations alike.
+
+Many of the issues responsible for slow growth and impracticality in the VCM, as well as its inability to scale, are either mitigated or solved completely by the transition to a fast, energy-efficient DCM and the applications built on it.
+
+| **Traditional Finance**                                                                                       																				   							 | **Regenerative Finance**                                                                                 																	       							  |

```

Comment:
in the interests of shortening the page, I suggest removing the table altogether - it's mostly repetition of the bullet points immediately preceding it